ldes-server-port-publication-rest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/converters/LdesFragmentConverterImplTest.java

```diff
@@ -1,64 +0,0 @@
-package be.vlaanderen.informatievlaanderen.ldes.server.rest.converters;
-
-import be.vlaanderen.informatievlaanderen.ldes.server.domain.config.LdesFragmentConfig;
-import be.vlaanderen.informatievlaanderen.ldes.server.domain.entities.LdesFragment;
-import be.vlaanderen.informatievlaanderen.ldes.server.domain.entities.LdesMember;
-import org.apache.jena.rdf.model.Model;
-import org.apache.jena.riot.Lang;
-import org.apache.jena.riot.RDFFormat;
-import org.apache.jena.riot.RDFParserBuilder;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
-
-import java.io.File;
-import java.io.IOException;
-import java.net.URISyntaxException;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.util.List;
-import java.util.Objects;
-import java.util.stream.Collectors;
-
-import static be.vlaanderen.informatievlaanderen.ldes.server.rest.converters.LdesFragmentConverter.outputLdesFragment;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-class LdesFragmentConverterImplTest {
-
-    private final LdesFragmentConfig ldesFragmentConfig = new LdesFragmentConfig();
-
-    @BeforeEach
-    void setup() {
-        ldesFragmentConfig.setView("http://localhost:8089/exampleData");
-        ldesFragmentConfig.setShape("http://localhost:8089/exampleData/shape");
-    }
-
-    @Test
-    @DisplayName("Correct conversion of LdesMember to JSONArray")
-    void when_LdesMemberIsProvided_LdesMemberConverterReturnsJsonArray() throws URISyntaxException, IOException {
-        LdesMember ldesMember = readLdesMemberFromFile("example-ldes-member.nq");
-        LdesFragment ldesFragment = new LdesFragment(List.of(ldesMember), ldesFragmentConfig.toMap());
-        String fragmentString = outputLdesFragment(ldesFragment, RDFFormat.NQUADS);
-
-        String expectedFragment = getContentFromFile("example-ldes-fragment.nq");
```

Comment:
I understand these tests are not relevant since LdesFragmentConverter is removed, but should we still not have a similar test to verify the fragment?